scripts/setup-wsl.ps1

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) Gilbertsoft LLC. All rights reserved.
```

Comment:
Please don't use a different license, thanks. Repo is Apache, https://github.com/drud/ddev/blob/master/LICENSE

Reply:
Sorry, just copied and pasted ;)